### PR TITLE
removed obsolete master_account reference

### PR DESCRIPTION
**Pull request by AngelBlock**

### CODE REVIEW

**Comment by telezhnaya** [comment]
As I understand, that's the fix of all comments in `lockup-factory` from https://hackmd.io/@ailisp/rJbiDYr5_
@ailisp adding you to the reviewers here, hope you are OK with that 🙂

**Review by ailisp — APPROVED** [review]
Looks good

File: lockup-factory/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "lockup-factory"
-version = "0.2.0"
```
Comment (telezhnaya) [inline lockup-factory/Cargo.toml:3]: Please bump the version after we will finally release the `lockup` contract

Comment (AngelBlock) [reply lockup-factory/Cargo.toml:3]: Already bumped in here to 0.2.1 https://github.com/near/core-contracts/pull/158/commits/862c3a1b28f92184382801612ea0e703c8a06e14

**Comment by telezhnaya** [comment]
Merged in https://github.com/near/core-contracts/pull/170

**Pull request closed without merging**